scripts/migration.sh

```diff
@@ -34,6 +34,12 @@ printf "Generating migration boilerplate for patch level $NEW_LEVEL..."
 
 echo "SET NAMES utf8mb4 COLLATE utf8mb4_bin;\n" > "$FWD_SCHEMA"
 echo "-- TODO: Implement your forward migration here\n" >> "$FWD_SCHEMA"
+
+echo "SELECT @patchlevel := value FROM `dbMetadata` WHERE `name` = `schema-patch-level`;\n">> "$FWD_SCHEMA"
+echo "IF @patchlevel != $PREV_LEVEL THEN\n">> "$FWD_SCHEMA"
```

Comment:
@hritvi, what happens when you run `node bin/db_patcher` after generating a migration with this script? Looking at this line I'd expect it to fail, because `$PREV_LEVEL` isn't quoted. (but I might be wrong)